Dockerfile

```diff
@@ -24,16 +24,16 @@ WORKDIR /code
 
```

Comment:
I understand perseo-main/.DS_Store should not be included in this PR. Maybe it should be added to .gitignore?

Reply:
Fixed in [ccd0caf](https://github.com/telefonicaid/perseo-core/pull/143/commits/ccd0caff61eda1b5cc612365e9d02e766f3fd21e)